native/build.gradle

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Reply:
I modified the existing `build.gradle` after moving it, which is why I jept the original header. Should I proceed with changing the header?

Reply:
@nipunayf github diff shows it as a newly added file (instead of moved file). lets keep as is then